composer.json

```diff
@@ -36,36 +38,42 @@
             "Common\\": "vendor/olcs/olcs-common/Common/src/Common"
         }
     },
+  "autoload-dev": {
+    "psr-4": {
+      "OlcsTest\\": "test/Olcs/src",
+      "PermitsTest\\": "test/Permits/src"
+    }
+  },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "config": {
-        "preferred-install": "dist",
-        "secure-http": false,
-        "platform": {
-            "php": "7.4.33"
-        },
         "sort-packages": true,
         "allow-plugins": {
-            "laminas/laminas-component-installer": true,
+            "bamarni/composer-bin-plugin": true,
             "laminas/laminas-dependency-plugin": true,
-            "elendev/composer-push": true,
-            "phpstan/extension-installer": true
+            "elendev/composer-push": true
```

Comment:
We can get rid of: `elendev/composer-push` from `allow-plugins` now I believe.

Reply:
removed

Reply:
done